arrows/gdal/tests/test_image.cxx

```diff
@@ -0,0 +1,124 @@
+/*ckwg +29
+ * Copyright 2013-2017 by Kitware, Inc.
+ * All rights reserved.
+ *
+ * Redistribution and use in source and binary forms, with or without
+ * modification, are permitted provided that the following conditions are met:
+ *
+ *  * Redistributions of source code must retain the above copyright notice,
+ *    this list of conditions and the following disclaimer.
+ *
+ *  * Redistributions in binary form must reproduce the above copyright notice,
+ *    this list of conditions and the following disclaimer in the documentation
+ *    and/or other materials provided with the distribution.
+ *
+ *  * Neither name of Kitware, Inc. nor the names of any contributors may be used
+ *    to endorse or promote products derived from this software without specific
+ *    prior written permission.
+ *
+ * THIS SOFTWARE IS PROVIDED BY THE COPYRIGHT HOLDERS AND CONTRIBUTORS ``AS IS''
+ * AND ANY EXPRESS OR IMPLIED WARRANTIES, INCLUDING, BUT NOT LIMITED TO, THE
+ * IMPLIED WARRANTIES OF MERCHANTABILITY AND FITNESS FOR A PARTICULAR PURPOSE
+ * ARE DISCLAIMED. IN NO EVENT SHALL THE AUTHORS OR CONTRIBUTORS BE LIABLE FOR
+ * ANY DIRECT, INDIRECT, INCIDENTAL, SPECIAL, EXEMPLARY, OR CONSEQUENTIAL
+ * DAMAGES (INCLUDING, BUT NOT LIMITED TO, PROCUREMENT OF SUBSTITUTE GOODS OR
+ * SERVICES; LOSS OF USE, DATA, OR PROFITS; OR BUSINESS INTERRUPTION) HOWEVER
+ * CAUSED AND ON ANY THEORY OF LIABILITY, WHETHER IN CONTRACT, STRICT LIABILITY,
+ * OR TORT (INCLUDING NEGLIGENCE OR OTHERWISE) ARISING IN ANY WAY OUT OF THE USE
+ * OF THIS SOFTWARE, EVEN IF ADVISED OF THE POSSIBILITY OF SUCH DAMAGE.
+ */
+
+/**
+ * \file
+ * \brief test GDAL image class
+ */
+
+#include <test_gtest.h>
+
+#include <arrows/gdal/image_io.h>
+#include <vital/plugin_loader/plugin_manager.h>
+#include <vital/types/metadata_traits.h>
+
+kwiver::vital::path_t g_data_dir;
+
+namespace algo = kwiver::vital::algo;
+namespace gdal = kwiver::arrows::gdal;
+static std::string nitf_file_name = "test.tif";
```

Comment:
It's probably worth expanding this test to load a couple of combinations of images of different data types and file formats.  For example, an 8-bit JPEG, a 16-bit Tiff, a float NITF.  I'm not sure we can encode the RPC metadata in JPEG, but that's okay.  The main point is can it open the different file types and get the correct values in the correct types.

Reply:
Done. I had to make the Tiff and NITF singe channel. The JPEG image is RBG. No metadata with the JPEG file.